hoomd/md/pytest/test_pair.py

```diff
@@ -0,0 +1,939 @@
+    invalid_params_list.append(_make_invalid_params(buckingham_valid_dict,
+                                                    buckingham_invalid_dicts,
+                                                    hoomd.md.pair.Buckingham,
+                                                    {}, "Buckingham"))
+
+    lj1208_valid_dict = {"sigma": 0.5, "epsilon": 0.0005}
+    lj1208_invalid_dicts = _make_invalid_param_dict(lj1208_valid_dict)
+    invalid_params_list.append(_make_invalid_params(lj1208_valid_dict,
+                                                    lj1208_invalid_dicts,
+                                                    hoomd.md.pair.LJ1208,
+                                                    {}, "LJ1208"))
+
+    fourier_valid_dict = {"a": [0.5, 1.0, 1.5], "b": [0.25, 0.034, 0.76]}
+    fourier_invalid_dicts = _make_invalid_param_dict(fourier_valid_dict)
+    invalid_params_list.append(_make_invalid_params(fourier_valid_dict,
+                                                    fourier_invalid_dicts,
+                                                    hoomd.md.pair.Fourier,
+                                                    {}, "Fourier"))
+
+    slj_valid_dict = {"sigma": 0.5, "epsilon": 0.0005}
+    slj_invalid_dicts = _make_invalid_param_dict(slj_valid_dict)
+    invalid_params_list.append(_make_invalid_params(slj_valid_dict,
+                                                    slj_invalid_dicts,
+                                                    hoomd.md.pair.SLJ,
+                                                    {}, "SLJ"))
+
+    dpd_valid_dict = {"A": 0.5, "gamma": 0.0005}
+    dpd_invalid_dicts = _make_invalid_param_dict(dpd_valid_dict)
+    invalid_params_list.append(_make_invalid_params(dpd_valid_dict,
+                                                    dpd_invalid_dicts,
+                                                    hoomd.md.pair.DPD,
+                                                    {'kT': 2}, "DPD"))
+
+    dpdlj_valid_dict = {'sigma': 0.5, 'epsilon': 0.0005, 'gamma': 0.034}
+    dpdlj_invalid_dicts = _make_invalid_param_dict(dpdlj_valid_dict)
+    invalid_params_list.append(_make_invalid_params(dpdlj_valid_dict,
+                                                    dpdlj_invalid_dicts,
+                                                    hoomd.md.pair.DPDLJ,
+                                                    {'kT': 1}, "DPDLJ"))
+
+    dlvo_valid_dict = {'kappa': 1.0, 'Z': 0.1, 'A': 0.1}
+    dlvo_invalid_dicts = _make_invalid_param_dict(dlvo_valid_dict)
+    invalid_params_list.append(_make_invalid_params(dlvo_valid_dict,
+                                                    dlvo_invalid_dicts,
+                                                    hoomd.md.pair.DLVO,
+                                                    {}, "DLVO"))
+    return [params for param_list in invalid_params_list for params in param_list]
+
+
+@pytest.fixture(scope="function", params=_invalid_params(), ids=(lambda x: x[0]))
```

Comment:
Multiple will have the same id then, yes? Maybe we should append a number?

Reply:
Also is there a reason we go from a list of valid parameters with everything else constant, to a list of tuples with everything in them? I think it would be more consistent to have a tuple with a list of parameters (then the id doesn't matter any more). We do parameterize more by using a list of tuples, but I don't know if we get much from that.

Reply:
Pytest will automatically append a number, we don't need to do that explicitly.